src/hotspot/share/opto/superword.cpp

```diff
@@ -1526,60 +1530,191 @@ void SuperWord::combine_pairs_to_longer_packs() {
 #endif
 }
 
-void SuperWord::split_packs_longer_than_max_vector_size() {
```

Comment:
Note: This is not covered by `split_packs_only_implemented_with_smaller_size`. No vector larger than max vector size is implemented, so we automatically split it down.